app/serializers/motion_serializer.rb

```diff
@@ -18,6 +17,7 @@ class MotionSerializer < ActiveModel::Serializer
 
   has_one :author, serializer: UserSerializer, root: 'users'
   has_one :outcome_author, serializer: UserSerializer, root: 'users'
+  has_one :discussion
```

Comment:
I'd like to not include this all the time, since it'll slow down a bunch of things, and make our json payloads much bigger (since including the discussion also includes the group, includes the author, etc.). The way we can include this conditionally:

```
has_one :discussion

def include_discussion?
  scope[:include_discussion]
end

def scope
  super || {} # (this simply ensures that the scope is not nil)
end
```
And in the controller:
```
respond_with_collection scope: { include_discussion: true }
```